test/performance/README.md

```diff
@@ -1,128 +1,189 @@
-## Stress Test Experiments
+## Workload Benchmark for KubeStellar
 
+*Pre-requisite*: in order to follow the instructions below, you must have an environment with KubeStellar installed; see [KubeStellar getting started](https://docs.kubestellar.io/release-0.23.1/direct/get-started/). Alternatively, you can also use KubeStellar e2e script [run-test.sh](https://github.com/kubestellar/kubestellar/blob/main/test/e2e/run-test.sh) to setup an environment.
 
-## Generate the workload traffic
+To generate the sample workload for KubeStellar performance experiments proceed as following:
 
-1. Clone the following fork of the clusterloader2 repo: 
+### a) Generate the workload traffic
 
-```bash
-$ git clone -b ks-provider https://github.com/dumb0002/perf-tests.git
-```
+1. Clone the following clusterloader2 repo: 
 
-2. Configure the parameters of your experiment:  
+   ```bash
+   $ git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+   ```
+
+   Set the variable `WORK_DIR` to the path of the subdirectory `clusterloader2/` of the cloned repo. For example: 
+
+   ```bash
+   $ export WORK_DIR=$HOME/perf-test/clusterloader2
```

Comment:
FYI: that's a pretty generic variable name. Maybe something more specific, such as `CL2_DIR`, would be better?

Reply:
Agree. Fixed!